brands/shop/beauty.json

```diff
@@ -174,5 +174,15 @@
       "name": "Yves Rocher",
       "shop": "beauty"
     }
+  },
+  "shop/beauty| Carlance": {
```

Comment:
no space after |

Reply:
hello, thanks for the feedback!

I have added my email and name, thank you.

so for this change do i need to again branch and create a new PR?
I think I forgot to run  "npm run build"